tutorials/katacoda/thanos/7-multi-tenancy/courseBase.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 docker pull quay.io/prometheus/prometheus:v2.20.0
-docker pull quay.io/thanos/thanos:v0.16.0-rc.1
+docker pull quay.io/thanos/thanos:v0.17.0-rc.0
```

Comment:
It feels premature to do this before the image is built & tested. This will make those tutorials to be updated sooner than image is built potentially ;p (not mentioning testing all)

Reply:
Right, this is only on the release branch though. I'll revert anyway then. :)

Reply:
Ah you are right. Then for it's up to us  (: